src/tex/appendix.tex

```diff
@@ -622,8 +622,24 @@ \subsection{Impact of sampling temperature}
+
+One can observe a clear progression of performance across model families (e.g., \ClaudeThreeFiveSonnet >  \ClaudeThree > \ClaudeTwo or \GPTFourO > \GPTFour > \GPTThreeFiveTurboZeroT). Newer versions consistently outperform their predecessors.
+Scores on knowledge-intensive tasks are typically lower than those on calculation and reasoning-intensive questions. Performance on reasoning-intensive questions in \chembench tends to be the highest.
+
+Based on these findings, some scope for improving these models could be to focus on enhancing knowledge-based training, possibly through improved pre-training on chemistry-specific texts or with chemistry knowledge-base integration. However, innovation is also needed to incorporate this information into the systems. For example, even \PaperQATwo could not outperform the \gls{llm} it is using for directing the tools, even though the agent has access to literature evidence.
+This suggests we might need to not only focus on building chemistry-specific retrieval datasets since current systems fail to retrieve the relevant systems and should be coupled with more domain-specific databases. However, the observation that our ReAct agents were too fragile to answer questions for which they had custom-made tools available correctly suggests that we also must invest in building more robust agent frameworks. 
+
+Models show the strongest performance in reasoning tasks on the \chembench corpus, which suggests that current training approaches are good at developing logical/reasoning capabilities for basic problem-solving of university-level exams. However, performance drops significantly for advanced tasks, suggesting that more advanced chemistry problems with complex, multi-step solutions (e.g., solving analytical chemistry problems) should be included in training or finetuning.
 
+The post-training (e.g., \gls{rlhf}) we currently do does not equip models with human-like \enquote{chemical taste.} This, however, will be essential for future discovery research (e.g., in combination with genetic algorithms).
 
+As commented above, \PaperQATwo was outperformed by the \gls{llm} powering it. Similarly, \ClaudeThreeFiveSonnet only slightly improved the results when equipped with some specific tools, while \GPTFourO performance decreased in this same tool environment (see \Cref{sec:react-environment} for further discussion about the ReAct environment). Therefore, a robust chemical agent environment optimized to solve general chemical problems, instead of focusing on bibliographic search or lab work, can be useful in the future.
```

Comment:
```suggestion
As commented above, \PaperQATwo was outperformed by the \gls{llm} powering it. Similarly, \ClaudeThreeFiveSonnet only slightly improved the results when equipped with some specific tools, while \GPTFourO performance decreased in this same tool environment (see \Cref{sec:react-environment} for further discussion about the ReAct environment). Therefore, a robust chemical agent framework optimized to solve general chemical problems instead of focusing on bibliographic search or lab work can be useful in the future.
```

in agent world being like to use language from reinforcement learning. In there, an environment is the environment in which the agent operates (e.g. the stuff it can interact with) - which is different from the framework driving the agent. 

in general - is this section not a bit redundant with the one above where you discuss agents? Perhaps it makes sense to put the two paragraphs together?

Reply:
@MrtinoRG added this bit, so if it appears somewhere else it can ne dropped?

Reply:
Now should be solved, I just removed the paragraph and added the reference to the appendix above